TTS/config/shared_configs.py

```diff
@@ -210,6 +210,13 @@ class BaseTrainingConfig(Coqpit):
         tb_model_param_stats (bool):
             Enable / Disable logging internal model stats for model diagnostic. It might be useful for model debugging.
             Defaults to ```False```.
+            wandb_disabled: bool = False
+        wandb_project_name (str):
+            Name of the W&B project. Defaults to config.model
+        wandb_entity (str):
```

Comment:
What does it do in practice?

Reply:
I assume you're referring to `wandb_enitity`. Entity is like a team. If you set an entity, all your runs will be logged in a team's dashboard where the member can view your run, multiple members can log runs in the same project as long as they set the same entity. They can read each other's reports etc. It's meant for collaboration.

Reply:
Thanks again :) 

What happens if you don't define it ?

Reply:
If it is set to `None`, the runs will be logged in your personal account and not the teams account